go/analysis/passes/printf/printf.go

```diff
@@ -66,6 +68,51 @@ argument list. Otherwise it is assumed to be Print-like, taking a list
 of arguments with no format string.
 `
 
+type PrintKind int
+
+const (
+	// Not a print or printf func or wrapper
+	KindNo PrintKind = iota
```

Comment:
KindNone Kind = iota // not a fmt wrapper function
KindPrint // function behaves like fmt.Print
KindPrintf //  function behaves like fmt.Printf